src/ansys/fluent/core/solver/flobject.py

```diff
@@ -150,6 +150,14 @@ def get_attr(self, attr) -> Any:
             raise RuntimeError("Object is not active")
         return attrs[attr] if attrs else None
 
+    def arguments(self) -> Any:
+        attrs = self.get_attrs(["arguments"])
+        if attrs:
+            attrs = attrs.get("attrs", attrs)
+        if "arguments" != "active?" and attrs and attrs.get("active?", True) is False:
```

Comment:
`"arguments" != "active?"` part is redundant

Reply:
Ok. Thank you.

Reply:
Done.